src/components/ChannelList/__tests__/ChannelList.test.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
```

Comment:
### Tests

✓ should re-query channels when filters change (233ms)
✓ should render `LoadingErrorIndicator` when queryChannels api throws error (27ms)
✓ when queryChannels api returns no channels, `EmptyStateIndicator` should be rendered (22ms)
- mobile navigation
    ✓ should call `closeMobileNav` prop function, when clicked outside ChannelList (573ms)
    ✓ should not call `closeMobileNav` prop function on click, if ChannelList is collapsed (226ms)
- Default and custom active channel
    ✓ should call `setActiveChannel` prop function with first channel as param (64ms)
    ✓ should call `setActiveChannel` prop function with channel (which has `customActiveChannel` id)  as param (48ms)
    ✓ should render channel with id `customActiveChannel` at top of the list (217ms)
- Event handling
    - message.new
    ✓ should move channel to top of the list (287ms)
    ✓ should not alter order if `lockChannelOrder` prop is true (360ms)
    - notification.message_new
    ✓ should move channel to top of the list by default (250ms)
    ✓ should call `onMessageNew` function prop, if provided (183ms)
    - notification.added_to_channel
    ✓ should move channel to top of the list by default (308ms)
    ✓ should call `onAddedToChannel` function prop, if provided (269ms)
    - notification.removed_from_channel
    ✓ should remove the channel from list by default (275ms)
    ✓ should call `onRemovedFromChannel` function prop, if provided (307ms)
    - channel.updated
    ✓ should update the channel in list, by default (157ms)
    ✓ should call `onChannelUpdated` function prop, if provided (169ms)
    - channel.deleted
    ✓ should remove channel from list, by default (179ms)
    ✓ should call `onChannelDeleted` function prop, if provided (145ms)
    ✓ should unset activeChannel if it was deleted (150ms)
    - connection.recovered
    ✓ should rerender the list (183ms)
    - channel.truncated
    ✓ should remove latest message (233ms)
    ✓ should call `onChannelTruncated` function prop, if provided (161ms)